### ISSUE TITLE
Mocs Arcade?

### ISSUE THREAD

**Issue by kingoftheconnors** [Jragoon/Andromeda#1]
Awwwwwww yeah!

I assume this is your final for Tanis' video game class? Congrats!

How would you feel with putting it on the Mocs Arcade?

**Comment by zzzunes** [comment]
Yeah! I would love to put this on the Mocs Arcade. I'm not sure how I need to package it, but I would definitely like to put it together.

**Comment by kingoftheconnors** [comment]
Just package it with three things:
- [ ] A bat script that will open the game
- [ ] A 500x430 image of the game for the game-selector
- [ ] A text file with basic info
These will be placed with the launcher and that'll be it.

The size for most of my game-selector images are 504x434, but that was just laziness on my part. As long as the dimensions are anything close to that, it'll work.

For the bat script, BananaBomber's entry is like this:
```
@CD C:\bananabomber
@CALL java -Djava.library.path="natives/windows" -jar "bananabomber.jar"
```
It just runs command-line commands to open it. Bananabomber was compiled into a jar file, but the lwjgl library was kept external, hence the use of -D to import it.

The info file looks like this:
```
NAME:
Banana Bomber;
TITLE (game window title while playing):
BananaBomber;
DESCRIPTION:
A reinterpretation of the classic Bomberman vs. Mode. Banana Bomber is a 2-player game in with two monkeys, both equipped with an infinite amount of bombs.
Mine through the map, reach each other and be the last one surviving!;
```
And yes, those semicolons are the delimiters for this file. It allows newlines to be used in the description if you want some nice spacing (or ASCII art!)

**Comment by zzzunes** [comment]
Awesome! Thanks Connor, this is really helpful. I'll work on this tomorrow and let you know when I get it together.

**Comment by kingoftheconnors** [comment]
Hey Jared, Andrew and I are starting some mocs arcade launcher work for the summer, and I wanted to know if you could finish andromeda and send it our way? I know Andrew put you on the game-upload website. I'm right now making as many small games as possible to give the arcade some range.

**Comment by zzzunes** [comment]
Didn't forget about you- I tested it on my friends and they didn't have any problems, so I'm zipping it up and linking it now.
